src/components/React/ControlsPanel.js

```diff
@@ -45,12 +45,17 @@ export default function ControlPanel(props) {
               });
               // Change color map
               reduxDispatch(
-                changeColorMap(val === "salt" ? colorMaps[0] : colorMaps[1])
+                changeColorMap(
+                  val.value === "salt"
+                    ? colorMaps.find((m) => m.name === "Haline")
+                    : colorMaps.find((m) => m.name === "Thermal")
```

Comment:
I knew there had to be a way to do this without a hard coded index! I'll have to take a quick look at the method